src/content/en/2019/fonts.md

```diff
@@ -11,120 +11,194 @@ last_updated: 2019-11-04T12:00:00+00:00:00
-The median web page makes 4 web font requests. Even the 10th percentile requests one. The ubiquity of Web fonts is complete. It does seem interesting to me that web font requests seem to be pretty steady across desktop and mobile. I’m glad to see the recommendation to hide `@font-face` blocks inside of a `@media` query didn’t catch on (don’t do this).
+(Graph Title: Distribution of Fonts Per Page; Source: 06.09a)
 
-**06.34: Font requests per page**
+The median web page makes four web font requests. The pages with the most web fonts (in the 90th percentile) requested 9 and 10 web fonts on mobile and desktop, respectively.
 
-Why are there more requests for fonts made on mobile devices? Is it related to fewer typefaces available on mobile devices, which in turn means fewer `local()` hits in Google Fonts CSS? Actually we might be on to something here.
+(Graph Title: Font Requests Per Page; Source: 06.34)
 
-**06.09b: Web font families**
+It does seem quite interesting that web font requests seem to be pretty steady across desktop and mobile. I’m glad to see the recommendation to hide `@font-face` blocks inside of a `@media` query didn’t catch on (don’t get any ideas).
 
-- [ ] TODO *Zach’s note: Is this families per page? What is `httparchive.almanac.parsed_css`?*
+That said there are marginally more requests for fonts made on mobile devices! My hunch here is that fewer typefaces are available on mobile devices, which in turn means fewer `local()` hits in Google Fonts CSS, falling back to network requests for these.
 
-These results were a little surprising. I didn’t expect this to line up so well with the number of web font requests per page. Namely I didn’t expect so many unique font families! To me this suggests a few things might be happening here:
+### You Don’t Want To Win This Award
 
-- Authors are mapping web fonts that share the same typeface to unique font-family names. My intuition seems to think this is unlikely.
-- Authors are only using one web font from each font-family (this will be more common as variable font usage increases).
-- I’ve seen some foundries (typography.com) use multiple font-family names as a copy-prevention mechanism for web fonts but I don’t think this is very common.
+The award for the page that requests the most web fonts goes to a site that made 718 web font requests.
 
-**06.10: Italics**
+After diving into the code, all of those 718 requests are going to Google Fonts! It looks like a malfunctioning “Above the Page fold” optimization plugin for WordPress has gone rogue on this site and is requesting (DDoS-ing?) all the Google Fonts—oops!
 
-Only 29% of pages declare at least one italic font. This suggests a lot of faux-italic across the web, as web browsers will synthesize a fake bold version if one is not available for use. That means using elements with italic user agent styling (`em`, `i`, et al) will render faux-italic.
+Ironic that a performance optimization plugin can make your performance much worse! 😈
```

Comment:
No guarantee how emoji will be displayed to users so it may be best to omit it.
```suggestion
Ironic that a performance optimization plugin can make your performance much worse!
```